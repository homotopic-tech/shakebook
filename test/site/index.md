---
title: MySite
date: 2020-04-28
---
