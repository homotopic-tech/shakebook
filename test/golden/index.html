<html>
  <head profile="http://www.w3.org/2005/10/profile">
      <meta charset="UTF-8">
      <meta name="description" content="">
      <meta name="author" content="">
      <meta name=viewport content="width=device-width, initial-scale=1">
      <title>MySite</title>
      <link rel="icon" type="image/png" href="/images/logo.png">
      <link rel="stylesheet"
        href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
        crossorigin="anonymous">
      <link rel="stylesheet"
        href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
        crossorigin="anonymous">
      <link rel="stylesheet" href="/css/style.css">
      <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      <script
        src="https://code.jquery.com/jquery-3.4.1.min.js"
        integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
        crossorigin="anonymous"></script>
      <script
        src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
        crossorigin="anonymous"></script>
      <script
        src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
         crossorigin="anonymous"></script>
      <style>
        
      </style>
  </head>
  
  <body>
    <header>
      <nav class="navbar navbar-expand flex-column flex-md-row">
        <a href="/" class="navbar-brand">Blanky Site</a>
        <div class="td-navbar-nav-scroll ml-md-auto">
          <ul class="navbar-nav mt-2 mt-lg-0">
            <li class="nav-item mr-4 mb-2 mb-lg-0">
              <a href="/docs">Docs</a>
            </li>
            <li>
              <a href="/posts">Posts</a>
            </li>
          </ul>
      </nav>
    </header>
    <main>
      <div class="text-center">
      <h1>This page is called MySite</h1>
      <p>It was generated from the source index.md</p>
      <p>For the url /index.html</p>
      </div>
      <div class="container">
      <p>Here are the three most recent posts:</p>
      <ul>
        <li>
          <a href="/posts/quuz.html">The Quux In The Maw</a>
          <small>Posted on Monday, July 20, 2020 with tags:
            <a href="/posts/tags/quux">quux</a>
            <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
        <li>
          <a href="/posts/baz.html">The Baz In the Hills</a>
          <small>Posted on Sunday, June 28, 2020 with tags:
            <a href="/posts/tags/baz">baz</a>
            <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
        <li>
          <a href="/posts/bar.html">The Bar in the mountains</a>
          <small>Posted on Monday, June 01, 2020 with tags:
            <a href="/posts/tags/bar">bar</a>
            <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
      </ul>
      </p>
      </div>
    </main>
  </body>
</html>
